llm4pddl/utils.py

```diff
@@ -215,6 +215,20 @@ def minify_pddl_problem(problem: str) -> str:
     return new_problem
 
 
+def flatten_pddl_problem(problem: str) -> str:
+    """Flattens pddl problem.
+
+    You *MUST* only run this after minify_pddl_problem(). Otherwise, it
```

Comment:
if this is the case, why don't we just combine the two? i.e., put this new logic into minify (since flattening is really just further minifying)

Reply:
my thought process is that we might want to use only minify vs both to test this out. Or do you just want to go ahead and combine and only use both?

Reply:
gotcha. if we do want to test this out, why don't you still add this to minify, but do something like
```
if FLAGS.llm_prompt_flatten_pddl:
    < your new code>
```
and then we can toggle that flag via the command line. and if we discover later that it's always as good or better to flatten we can just remove the flag and unindent.

Reply:
gotcha

Reply:
yessir!